<?xml version="1.0" encoding="UTF-8"?>
<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
<!--                                                                                             -->
<!-- Copyright (c) 2015 The Regents of the University of California, Author: William Lenthe      -->
<!--                                                                                             -->
<!-- This program is free software: you can redistribute it and/or modify it under the terms of  -->
<!-- the GNU Lesser General Public License as published by the Free Software Foundation, either  -->
<!-- version 2 of the License, or (at your option) any later version.                            -->
<!--                                                                                             -->
<!-- This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;   -->
<!-- without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   -->
<!-- See the GNU Lesser General Public License for more details.                                 -->
<!--                                                                                             -->
<!-- You should have received a copy of the GNU Lesser General Public License                    -->
<!-- along with this program.  If not, see <http://www.gnu.org/licenses/>.                       -->
<!--                                                                                             -->
<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->

<ui version="4.0">
 <class>FloatVec4Widget</class>
 <widget class="QFrame" name="FloatVec4Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>53</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="aData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="bData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="cData"/>
   </item>
   <item>
    <widget class="QLineEdit" name="dData"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>aData</tabstop>
  <tabstop>bData</tabstop>
  <tabstop>cData</tabstop>
  <tabstop>dData</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
